core/architect_bot.py

```
@@ -140,9 +141,104 @@ async def handle_quick_data(message, say, context):
+                                        user_id=user,
+                                        research_type=None,
+                                        thread_id=ts,
+                                        channel_id=channel,
+                                        include_visualizations=True
+                                    )
+                                    await self._send_architect_results(say, result, channel, ts)
+                                except Exception as e:
+                                    logger.error(f"Architect request from audio transcript failed: {e}")
+                                    await self.app.client.chat_postMessage(channel=channel, text=f"❌ Architect agent failed: {str(e)}", thread_ts=ts)
+                            else:
+                                await self.app.client.chat_postMessage(channel=channel, text=f"❌ Audio transcription failed.", thread_ts=ts)
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)